src/ts/app/console/src/render.ts

```diff
@@ -0,0 +1,28 @@
+import {html, TemplateResult} from "lit-html"
+import {ConsoleFragment} from "./subtree"
+
+
+
+export function render(fragments: ConsoleFragment[]): TemplateResult {
+    return html`
+        <style>
+        
+        div {
+            width: 100%;
+            overflow: hidden;
+            white-space: pre;
+            font-family: monospace;
+        }
+        
+        .stdout {
+            color: green
+        }
+
+        .stderr {
+            color: orange
+        }
+        </style>
+        
+        <div>${fragments.map((f) => html`<span class="${f.type}">${f.text}</span>`)}</div>
```

Comment:
eh, I don't think seeing stdout, and stderr is useful to kids, would rather highlight white or red.

Reply:
Maybe not but surely it should be an option. Even if it's not the default.

All that's pending better default output choices (e.g. turtle, processing...).